### PR TITLE
JDP210201-3

### PR DESCRIPTION
**Pull request by wisniewskimateusz**

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/CartController.java [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+    }
+
+    @GetMapping(value = "getProducts")
+    public List<ProductDto> getProducts() {
+        return new ArrayList<>();
+    }
+
+    @PutMapping(value = "addToCart")
+    public void addProductsToCart(@RequestParam List<ProductDto> productsDto) {
+
```
Comment (Bauaser-kun) [inline src/main/java/com/kodilla/ecommercee/CartController.java:24]: Should be  ProductDto instead of List

**Review by Bauaser-kun — CHANGES_REQUESTED** [review]

File: src/main/java/com/kodilla/ecommercee/CartController.java (hunk 1 above)
Comment (wisniewskimateusz) [reply src/main/java/com/kodilla/ecommercee/CartController.java:24]: but this added many Products to Cart, and the isn't field quantity i Product class. So how to add many products (the same) to cart?

Comment (Ktubbo) [reply src/main/java/com/kodilla/ecommercee/CartController.java:24]: But where you want to build this list of product? Imagine online shop, you pick a product, choose quantity of it and press "add to cart". Then products are added to a cart. I think that you will need, cartId to get specific Cart, productId to add specific product to a Cart and quantity of a product which you want to add.

Comment (wisniewskimateusz) [reply src/main/java/com/kodilla/ecommercee/CartController.java:24]: `addProductToCart(@RequestParam Long cartId, @RequestParam Long productId)`

File: src/main/java/com/kodilla/ecommercee/CartController.java [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+    @PutMapping(value = "addToCart")
+    public void addProductToCart(@RequestParam Long cartId, @RequestParam Long productId) {
+
+    }
+
+    @DeleteMapping(value = "deleteProduct")
+    public void deleteProduct(@RequestParam Long productId) {
+
+    }
```
Comment (Ktubbo) [inline src/main/java/com/kodilla/ecommercee/CartController.java:30]: Same thing here, you need to indicate specific cart and delete specific product from it.

**Review by Ktubbo — CHANGES_REQUESTED** [review]

**Review by Bauaser-kun — APPROVED** [review]

**Review by Ktubbo — APPROVED** [review]

**Pull request merged**